README.md

```diff
@@ -1,2 +1,70 @@
-# cast-le-frontend
-The frontend repository for the cast-le project
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
A lot of the README that is autogenerated when you set up the app isn't relevant. How would we feel about rewriting/filtering most of it to leave only the pieces we feel would be most relevant to our developers?

Reply:
Can do this for sure, do we have a features list/ slogan, maybe a oneliner about the app? The rest, I could prob come up with.

Reply:
Not yet, so this can def be filled out as we go, but just having what anybody would need to get started is more than plenty. Some examples you could refer to:
- [bryti's](https://github.com/passiondynamics/bryti) (even here, I still need to fill out more info before `Contributing`, but barebones isn't a bad thing)
- [the backend's](https://github.com/passiondynamics/cineguess-backend) (this one has even less but same thing)
- [overseerr's](https://github.com/sct/overseerr) (a non-backend-only example! Should look a little familiar, but an example for what a frontend-ish repo might showcase more)
- (I'll think of some more)